Libs/MRML/Core/vtkMRMLColorTableNode.cxx

```diff
@@ -168,7 +168,10 @@ void vtkMRMLColorTableNode::Copy(vtkMRMLNode *anode)
     if (this->LookupTable == nullptr)
     {
       vtkNew<vtkLookupTable> lut;
-      this->SetAndObserveLookupTable(lut.GetPointer());
+      // Only copy the RGBA values and not modify the defined state of colors
+      // (all the color properties are copied in Superclass::Copy).
+      const bool setAllColorsToDefined = false;
```

Comment:
The value associated with variable `setAllColorsToDefined` seems to contradict 

> This commit changes the default behavior of `SetAndObserveLookupTable` so that it sets all the colors in the lookup table to "defined".